src/components/docPage/index.tsx

```diff
@@ -108,6 +112,34 @@ export function DocPage({
           )}
         </main>
       </section>
+      {/* can't use useEffect here (server component) */}
+      <Script
+        type="module"
+        id="mermaid-script"
+        dangerouslySetInnerHTML={{
+          __html: javascript`
+              const mermaidBlocks = document.querySelectorAll('.language-mermaid');
+              if (mermaidBlocks.length >= 0) {
+                const {default: mermaid} = await import(
+                  'https://cdn.jsdelivr.net/npm/mermaid@11/dist/mermaid.esm.min.mjs'
```

Comment:
this breaks our docs CSP, you'll need to add this script url to our vercel.json

Reply:
better yet add it as a dependency 👍